index.html

```diff
@@ -75,15 +143,33 @@ <h1>Mouse test</h1>
     });
 
     document.addEventListener("mousedown", event => {
-        switch (event.button) {
-            case 0: circle.classList.add("active-left"); break;
-            case 1: circle.classList.add("active-middle"); break;
-            case 2: circle.classList.add("active-right"); break;
-        }
+        circle.classList.add(`active-${event.button}`);
+
+        event.preventDefault();
+
+        const diff = performance.now() - lastClick[event.button];
+        const bad = diff < 200;
```

Comment:
What's the use case here? Why a delay greater than 200ms is considered "bad"?

Reply:
I'll rename it to highlight, which is essentially what it is, to easier catch quick clicks at a glance (for broken mice).